Resources/Textures/Objects/Weapons/Bombs/spidercharge.rsi/meta.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": 1,
```

Comment:
the indentation here isn't correct. consult other meta.json files for the correct styling.